packages/zoe/src/typeGuards.js

```diff
@@ -79,3 +122,219 @@ export const InvitationElementShape = M.splitRecord({
+    M.remotable('zcfSeat'),
+  ).returns(),
+});
+
+export const ExitObjectI = M.interface('Exit Object', {
+  exit: M.call().returns(),
+});
+
+export const InstanceAdminI = M.interface('InstanceAdmin', {
+  makeInvitation: M.call(InvitationHandleShape, M.string())
+    .optional(M.any(), M.pattern())
+    .returns(M.promise()),
+  saveIssuer: M.call(M.eref(IssuerShape), M.string()).returns(M.promise()),
+  makeZoeMint: M.call(KeywordShape)
+    .optional(AssetKindShape, DisplayInfoShape, M.pattern())
+    .returns(M.remotable('zoeMint')),
+  registerFeeMint: M.call(KeywordShape, M.remotable('feeMintAccess')).returns(
+    M.remotable('feeMint'),
+  ),
+  makeNoEscrowSeatKit: M.call(
+    AmountKeywordRecordShape,
+    ProposalShape,
+    M.remotable('ExitObj'),
+    SeatShape,
+  ).returns({ userSeat: SeatShape }),
+  replaceAllocations: M.call(SeatHandleAllocations).returns(),
+  exitAllSeats: M.call(M.any()).returns(),
+  failAllSeats: M.call(M.any()).returns(),
+  exitSeat: M.call(SeatShape, M.any()).returns(),
+  failSeat: M.call(SeatShape, M.any()).returns(),
+  stopAcceptingOffers: M.call().returns(),
+  setOfferFilter: M.call(M.arrayOf(M.string())).returns(),
+  getOfferFilter: M.call().returns(M.string()),
+  getExitSubscriber: M.call(SeatShape).returns(SubscriberShape),
+});
+
+export const InstanceStorageManagerGuard = M.interface(
+  'InstanceStorageManager',
+  {
+    getTerms: M.call().returns(M.split(TermsShape, M.record())),
+    getIssuers: M.call().returns(IssuerKeywordRecordShape),
+    getBrands: M.call().returns(BrandKeywordRecordShape),
+    getInstallationForInstance: M.call().returns(InstallationShape),
+    getInvitationIssuer: M.call().returns(IssuerShape),
+
+    saveIssuer: M.call(IssuerShape, M.string()).returns(M.promise()),
+    makeZoeMint: M.call(KeywordShape)
+      .optional(AssetKindShape, DisplayInfoShape, M.pattern())
+      .returns(M.or(ZoeMintShape, M.remotable('zoeMint'), M.promise())),
+    registerFeeMint: M.call(KeywordShape, M.remotable('feeMintAccess')).returns(
+      IssuerKitShape,
+    ),
+    getInstanceRecord: M.call().returns(InstanceRecordShape),
+    getIssuerRecords: M.call().returns(M.arrayOf(IssuerRecordShape)),
+    getWithdrawPayments: M.call().returns(
+      M.interface('WithdrawFacet', {
+        withdrawPayments: M.call(AmountKeywordRecordShape).returns(
+          PaymentKeywordRecordShape,
+        ),
+      }),
+    ),
+    initInstanceAdmin: M.call(
+      InstanceHandleShape,
+      M.remotable('instanceAdmin'),
+    ).returns(M.promise()),
+    deleteInstanceAdmin: M.call(InstanceAdminI).returns(),
+    makeInvitation: M.call(InvitationHandleShape, M.string())
+      .optional(M.any(), M.pattern())
+      .returns(M.promise()),
+    getRoot: M.call().returns(M.promise()),
+    getAdminNode: M.call().returns(M.remotable('adminNode')),
+  },
+);
+
+// In `SwingSet/src/types-external.js`, it's typed as `*`.
+const BundleCapShape = M.any();
+
+export const ZoeStorageMangerI = {
```

Comment:
This looks like a kit of interfaces, rather than an interface. Also, beware unprotected object literals.
```suggestion
export const ZoeStorageMangerKit = harden({
```

Reply:
This directly relates to https://github.com/Agoric/agoric-sdk/pull/6502#discussion_r1014543291 , and I think the current state of that discussion would suggest `ZoeStorageManagerIKit`.

Also note the "Manger" typo (should be "Manager").

Reply:
done.  pervasively added IKit suffixes where required.